sdk/dotnet/sdk/global.json

```diff
@@ -1,5 +1,9 @@
 ﻿{
   "msbuild-sdks": {
-    "MSTest.Sdk": "3.4.0"
+    "MSTest.Sdk": "3.7.0"
+  },
+  "sdk": {
+    "version": "8.0.101",
```

Comment:
It would be better to use `8.0.404` to reflect the latest version from Microsoft.

Reply:
Done